### DISCUSSION TITLE
Power connections between racks?

### DISCUSSION THREAD

**Discussion by calvaradESU** [opendcim/openDCIM#1404] [Q&A] [answered]
Is it possible to do a power connection from one rack to another?

**Comment by samilliken** [comment, marked as answer]
No, and that’s on purpose.

 

Scott

 

From: calvaradESU 
Sent: Wednesday, July 26, 2023 5:45 PM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] Power connections between racks? (Discussion #1404)

 

Is it possible to do a power connection from one rack to another?
